atomics/include/desul/atomics/Common.hpp

```diff
@@ -0,0 +1,49 @@
+/* 
+Copyright (c) 2019, Lawrence Livermore National Security, LLC
+and DESUL project contributors. See the COPYRIGHT file for details.
+Source: https://github.com/desul/desul
+
+SPDX-License-Identifier: (BSD-3-Clause)
+*/
+
+#ifndef DESUL_ATOMICS_COMMON_HPP_
+#define DESUL_ATOMICS_COMMON_HPP_
+#include "Macros.hpp"
```

Comment:
Why do you prefer `#include "foo.hpp"` over `#include <atomics/foo.hpp>`?

Reply:
Generally I also would like to see "desul" somewhere in the include lines. It just feels "atomics/Compare_Exchange.hpp" is a bit generic.

Reply:
That was partially a mistake, I'm good with `"desul/atomics/foo.hpp"`, and will make that change shortly, but it isn't a system header or an external dependence, so I have a preference for using the more self-documenting non-system include with quotes rather than angle brackets.

Reply:
Actually, the reasoning for the bare "foo.hpp" was that is an unambiguous local directory include, but either way, the path version is fine.